src/utils/validateVoucherCode.tsx

```diff
@@ -11,15 +12,36 @@ export const checkDuplicate = (
   return voucherCodes.includes(voucherCode);
 };
 
+export class LimitReachedError extends Error {
+  constructor(message: string) {
+    super(message);
+    this.name = "LimitReachedError";
+  }
+}
+
+export const checkLimitReached = (
+  lengthOfVoucherCodes: number,
+  limit = maxNumVouchersPerTransaction
+): boolean => {
+  return lengthOfVoucherCodes > limit;
+};
+
 export const validateVoucherCode = (
   voucherCode: string,
   voucherCodes: string[]
 ): boolean => {
+  const hasReachedLimit = checkLimitReached(voucherCodes.length + 1);
```

Comment:
just curious, any reason to use this method instead of having `lengthOfVoucherCodes >= limit` ?

Reply:
I've updated the code to use this instead, I initially did something similar, but somehow changed it to send in the length of the voucher codes 🤷